packages/logger/src/root-app-insights-logger-client.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as appInsights from 'applicationinsights';
+import { inject, injectable } from 'inversify';
+
+import { BaseAppInsightsLoggerClient } from './base-app-insights-logger-client';
+import { BaseTelemetryProperties } from './base-telemetry-properties';
+import { loggerTypes } from './logger-types';
+
+@injectable()
+export class RootAppInsightsLoggerClient extends BaseAppInsightsLoggerClient {
```

Comment:
RootAppInsightsLoggerClient is little ambiguous since it is extending Base what extra functionality it is adding is not clean from the name.

Reply:
how about using the word main?